### PR TITLE
feat(admin): group synchronization dialog now displays the interval b…

### PR DESCRIPTION
**Pull request by xflord**
…etween synchronizations

### CODE REVIEW

**Review by bodnara — CHANGES_REQUESTED** [review]
The interval is displayed properly, well done on that. However this dialog seems to be quite old and looks inconsistent with the reset of the app. I would like to ask you if you would be so kind and implement following changes:
image

- Change **all** the rows to be a input mat-form-field as shown in the screenshot (readonly), also remove ":" from the placeholder
- Change the order and color of buttons as per screenshot
- Remove redundant methods/elements that are in this component

File: libs/perun/dialogs/src/lib/group-sync-detail-dialog/group-sync-detail-dialog.component.ts [hunk 1]
```diff
@@ -96,26 +83,6 @@ export class GroupSyncDetailDialogComponent implements OnInit {
     return this.type === 'STRUCTURED';
   }
```
Comment (mattjoke) [inline libs/perun/dialogs/src/lib/group-sync-detail-dialog/group-sync-detail-dialog.component.ts:85]: I think `isBasic` and `isStructured` methods are not used. I think you should remove them

**Review by mattjoke — COMMENTED** [review]
Otherwise I think it looks good.

File: apps/admin-gui/src/app/vos/pages/group-detail-page/group-detail-page.component.html [hunk 2]
```diff
@@ -43,16 +43,12 @@
...
             , {{'GROUP_DETAIL.SYNCHRONIZATION' | translate}}:
             <span
               [matTooltipPosition]="'above'"
               [matTooltip]="group | groupSyncToolTip | translate">
-              <button
-                (click)="onSyncDetail()"
-                *ngIf="syncAuth"
-                [disabled]="(group | groupSyncIcon) === 'sync_disabled'"
-                mat-icon-button>
+              <button (click)="onSyncDetail()" mat-icon-button>
```
Comment (bodnara) [inline apps/admin-gui/src/app/vos/pages/group-detail-page/group-detail-page.component.html:51]: Here we lost the information if user is authorized to force group synchornization. We can move this check to the dialog itself so we can still open it but cannot perform the action.

**Review by bodnara — CHANGES_REQUESTED** [review]
There are couple of improvements:
image

- The synchronization field is not translated
- The dialog width is not set up, please set some reasonable width (450px maybe) when opening the dialog
- Right now the logic of sync and syncStructured is duplicated (2 buttons, 2 methods), I guess just merge it to one
- Last thing, I know the last sync time has the same format as before, but maybe we could use something more readable (this point more of a reference thing, if you think the format is fine than leave it as is)

**Comment by xflord** [comment]
> Last thing, I know the last sync time has the same format as before, but maybe we could use something more readable (this point is more of a preference thing, if you think the format is fine than leave it as is)

For now, I just removed the milliseconds and we can discuss it further tomorrow.
Fixed the rest

File: libs/perun/dialogs/src/lib/group-sync-detail-dialog/group-sync-detail-dialog.component.ts [hunk 3]
```diff
@@ -41,33 +40,21 @@ export class GroupSyncDetailDialogComponent implements OnInit {
...
-  }
-
   onCancel(): void {
     this.dialogRef.close(null);
   }
 
   onForce(): void {
     this.loading = true;
-    if (this.isBasic()) {
+    if (this.type === 'BASIC') {
```
Comment (HejdaJakub) [inline libs/perun/dialogs/src/lib/group-sync-detail-dialog/group-sync-detail-dialog.component.ts:49]: Again please replace the deprecated subscription by the recommended observer argument `{next: (), error: ()}`

**Review by HejdaJakub — COMMENTED** [review]
Just one suggestion, it looks good otherwise.

**Review by mattjoke — APPROVED** [review]
Looks good :+1:

**Review by bodnara — APPROVED** [review]

**Review by HejdaJakub — APPROVED** [review]

**Pull request merged**